### PR TITLE
Fix Cluster API client passing

### PR DESCRIPTION
**Pull request by lin007pro**

### CODE REVIEW

File: pkg/registration/registration_client.go [hunk 1]
```diff
@@ -112,6 +112,16 @@ func (rClient *K8sRegistrationClient) GetActionPolicy() []*proto.ActionPolicyDTO
 
 	rClient.addActionPolicy(ab, app, appPolicy)
 
+	// 4. node: move, provision; not resize;
+	node := proto.EntityDTO_VIRTUAL_MACHINE
+	nodePolicy := make(map[proto.ActionItemDTO_ActionType]proto.ActionPolicyDTO_ActionCapability)
+	nodePolicy[proto.ActionItemDTO_MOVE] = supported
```
Comment (enlinxu) [inline pkg/registration/registration_client.go]: kubeturbo is not handling node move, let's remove this line or change it to notsupported

File: pkg/registration/registration_client_test.go [hunk 2]
```diff
@@ -65,6 +66,12 @@ func TestK8sRegistrationClient_GetActionPolicy(t *testing.T) {
 	expected_app[provision] = recommend
 	expected_app[suspend] = recommend
 
+	expected_node := make(map[proto.ActionItemDTO_ActionType]proto.ActionPolicyDTO_ActionCapability)
+	expected_node[move] = supported
```
Comment (enlinxu) [inline pkg/registration/registration_client_test.go]: Same here, remove this line or change it to notsupported

**Comment by ading1977** [comment]
@lin007pro , I think what you are seeing is because you have a stitched environment, in which case, the actionDTO sent from the server contains the real VM name. However, if you only have a standalone kubeturbo target (i.e., remove the AWS target), you probably still need the original logic in order to find the machine from node. Can you please confirm?

Also, please add some description to this pull request, and update the title to correct the typo (passing -> parsing).

**Comment by lin007pro** [comment]
The DTO request contains the machine name, which is expected by the Cluster API.
Fix the passing of the Cluster API client down the line properly.

**Pull request merged**